tests/GUIUtils.lua

```diff
@@ -99,18 +99,21 @@ describe("Tests the GUI utilities as far as possible without mudlet", function()
         { "\27[38;2;120;134;94m", "<120,134,94:>"},
         { "\27[48;2;85;250;33m", "<:85,250,33>"},
         { "\27[38;2;120;134;94;48;2;85;250;33m", "<120,134,94:85,250,33>"},
```

Comment:
To add... What is meant by the 03/10 in the standard? I don't recognize the notation.